.github/workflows/e2e_additional_editor_and_player.yml

```diff
@@ -42,3 +45,9 @@ jobs:
         with:
           name: protractor-screenshots
           path: /home/runner/work/oppia/protractor-screenshots
+      - name: Uploading protractor-video as Artifacts
+        if: startsWith(github.head_ref, 'update-changelog-for-release') == false && always()
+        uses: actions/upload-artifact@v2
+        with:
+          name: protractor-video
+          path: /home/runner/work/oppia/protractor-video
```

Comment:
are we expecting it to be inside /work/ folder? (like the way we have done in circleCi)

Reply:
I believe this path is correct. GitHub actions installs the repo to `/home/runner/work/oppia/oppia`, so this `protractor-video` path will be in the same folder as `protractor-screenshots` and `oppia_tools`

Reply:
Ah, I see. Thanks for the explanation!